frontend/ui/widget/imageviewer.lua

```diff
@@ -602,7 +602,7 @@ function ImageViewer:onCloseWidget()
         self._images_list.free()
     end
     UIManager:setDirty(nil, function()
-        return "partial", self.main_frame.dimen
+        return "flashui", self.main_frame.dimen
```

Comment:
I'm satisfied with "ui", but just letting you know that ImageViewer can be launched from a CRE document when holding on an image. So, when closing, you may get back to book text (so, I guess in your logic, that may be flashpartial here, of flashui with no region, like quickdictlookup ?)

Reply:
Hmm, yeah, maybe, but it's used in so many other places that a 'no-region' approach might be a tad *too* destructive ;).

Theretically, I'm good with flashui, but good point, that's something to look out for, see if any kind of other use-cases might warrant a change ;).

Reply:
BTW, flashpartial wouldn't work here, because it's effectively a no-op on REAGL devices (i.e., it behaves like a "partial") ;).